src/main/java/net/glowstone/block/GlowBlock.java

```diff
@@ -452,5 +461,6 @@ public void applyPhysics(Material oldType, int newTypeId, byte oldData, byte new
         if (type != null) {
             type.onBlockChanged(this, oldType, oldData, newType, newData);
         }
+        type.receivePulse(this);
```

Comment:
Not what I expected. But it should work. Any reason we pulse even if applyPhysics is set to false ?